test/run-integration.js

```diff
@@ -330,13 +331,33 @@ function dnsFailThenSuccess(t) {
   });
 }
 
+function dnsFailThenSuccessNoSearch(t) {
+  t.plan(1);
+
+  var client_ = algoliasearch(
+    appId,
+    apiKey, {
+      // .biz is a black hole DNS name (not resolving)
+      hosts: [appId + '-dsn.algolia.biz', appId + '-dsn.algolia.net'],
+      timeout: 120000, // let's wait for the DNS timeout
```

Comment:
👍 even though this is not ES6. A good old var with caps can do the trick, right?